src/lib/components/tablev2/SingleSelectableContent.tsx

```diff
@@ -182,7 +183,12 @@ export function SingleSelectableContent<
           </HeadRow>
         ))}
       </div>
-      <TableBody role="rowgroup" className="tbody" ref={handleScrollbarWidth}>
+      <TableBody
+        role="rowgroup"
+        className="tbody"
+        ref={handleScrollbarWidth}
+        data-testid="table-body"
```

Comment:
I think this can be dropped

Reply:
The table header has a `div[role="row"]` as well, so I use this to target only table body's rows